content/hi/docs/concepts/architecture/cloud-controller.md

```diff
@@ -0,0 +1,204 @@
+
+## अधिकृतता
+
+इस खंड में बताया गया है कि क्लाउड नियंत्रक प्रबंधक को अपने ऑपरेशन करने के लिए विभिन्न API ऑब्जेक्ट्स पर कौन-कौन सी पहुंच की आवश्यकता होती है।
+
+### नोड नियंत्रक {#authorization-node-controller}
+
+नोड नियंत्रक केवल नोड ऑब्जेक्ट्स के साथ काम करता है। इसे नोड ऑब्जेक्ट्स को पढ़ने और संशोधित करने के लिए पूर्ण पहुंच की आवश्यकता है।
+
+`v1/Node`:
+
+- get
+- list
+- create
+- update
+- patch
+- watch
+- delete
+### रूट नियंत्रक {#authorization-route-controller}
+
+रूट नियंत्रक नोड ऑब्जेक्ट के निर्माण को सुनता है और सही रूट कॉन्फ़िगर करता है। इसे नोड ऑब्जेक्ट्स के लिए Get पहुंच की आवश्यकता होती है।
+
+`v1/Node`:
+
+- get
+
+### सेवा नियंत्रक {#authorization-service-controller}
+
+सेवा नियंत्रक सेवा ऑब्जेक्ट **निर्माण**, **अपडेट**, और **हटाने** के घटनाओं का प्रतीक्षा करता है और फिर
+उन सेवाओं के लिए उचित अंतक्षेत्र कॉन्फ़िगर करता है (EndpointSlices के लिए,
+kube-controller-manager इन्हें आवश्यकता के हिसाब से प्रबंधित करता है)।
+
+सेवाओं तक पहुंचने के लिए, इसे **सूची** और **देखो** पहुंच की आवश्यकता है। सेवाओं को अपडेट करने के लिए, इसे
+**पैच** और **अपडेट** पहुंच की आवश्यकता है।
+
+सेवाओं के लिए अंतक्षेत्र संसाधन सेटअप करने के लिए, इसे **निर्माण**, **सूची**, **प्राप्त करो**,
+**देखो**, और **अपडेट** का पहुंच चाहिए।
+
+`v1/Service`:
+
+- सूची
+- प्राप्त करो
+- देखो
+- पैच
+- अपडेट
+### अन्य {#authorization-miscellaneous}
+
+क्लाउड नियंत्रक प्रबंधक के मौलिक के कार्यान्वयन के लिए स्थापना को ईवेंट ऑब्जेक्ट बनाने की पहुंच की आवश्यकता है,
+और सुरक्षित प्रचालन सुनिश्चित करने के लिए, इसे सर्विस अकाउंट बनाने की पहुंच की आवश्यकता है।
+
+`v1/Event`:
+
+- create
+- patch
+- update
+
+`v1/ServiceAccount`:
+
+- create
+
+क्लाउड नियंत्रक प्रबंधक के लिए {{< glossary_tooltip term_id="rbac" text="RBAC" >}} ClusterRole इस तरह दिखता है:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cloud-controller-manager
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - '*'
+- apiGroups:
+  - ""
+  resources:
+  - nodes/status
+  verbs:
+  - patch
+- apiGroups:
+  - ""
+  resources:
+  - services
+  verbs:
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  verbs:
+  - create
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumes
+  verbs:
+  - get
+  - list
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
+  verbs:
+  - create
+  - get
+  - list
+  - watch
+  - update
+```
+## {{% heading "whatsnext" %}}
+
+* [क्लाउड नियंत्रक प्रबंधक प्रशासन](/docs/tasks/administer-cluster/running-cloud-controller/#cloud-controller-manager)
+  में क्लाउड नियंत्रक प्रबंधक को चलाने और प्रबंधित करने के निर्देश हैं।
+
+* एक एचए कंट्रोल प्लेन को क्लाउड नियंत्रक प्रबंधक का उपयोग करने के लिए अपग्रेड करने के लिए, देखें
```

Comment:
क्लाउड नियंत्रक प्रबंधक का उपयोग करने के लिए HA नियंत्रण प्लेन को अपग्रेड करने के लिए, देखें

I think this sentence looks good WDYT ?